src/components/va-alert/va-alert.tsx

```diff
@@ -144,7 +144,7 @@ export class VaAlert {
             aria-label={this.closeBtnAriaLabel}
             onClick={this.closeHandler.bind(this)}
           >
-            <i class="fa-times-circle" aria-label="Close icon" />
+            <i aria-hidden="true" class="fas fa-times-circle" role="presentation" />
```

Comment:
I don't think the `fas` class is doing anything, so it should be removed. What appear to be fontawesome classes are coming from here:

https://github.com/department-of-veterans-affairs/component-library/blob/cfffd5588044c7c2f37a5bae0be625aa50f8c1af/src/components/va-alert/va-alert.css#L141-L143

Reply:
Should be all good here! Thanks for the feedback.